sdks/python/apache_beam/utils/timestamp.py

```diff
@@ -269,6 +269,9 @@ def __mod__(self, other):
     other = Duration.of(other)
     return Duration(micros=self.micros % other.micros)
 
+  def __neg__(self):
+    return -self.micros
+
```

Comment:
I guess we're negating a timestamp somewhere and that was triggering invalid-unary-operand-type?

Should this return a Timestamp? e.g. `Timestamp(micros=-self.micros)`

Reply:
Done 👍